datadog_checks_dev/datadog_checks/dev/plugin/tox.py

```diff
@@ -51,15 +52,23 @@ def tox_configure(config):
 def add_style_checker(config, sections, make_envconfig, reader):
     # testenv:style
     section = '{}{}'.format(tox.config.testenvprefix, STYLE_CHECK_ENV_NAME)
+
+    # A list of files, plus any command line options accepted by mypy.
+    # See: https://mypy.readthedocs.io/en/stable/command_line.html
+    # Default to '-V' (i.e. 'show version') so that mypy exits immediately in the default case,
+    # instead of raising an error.
+    mypy_args = sections['testenv'].get(MYPY_ARGS, '-V')
+
     sections[section] = {
         'platform': 'linux|darwin|win32',
         # These tools require Python 3.6+
         # more info: https://github.com/ambv/black/issues/439#issuecomment-411429907
         'basepython': 'python3',
         'skip_install': 'true',
-        'deps': 'flake8\nflake8-bugbear\nflake8-logging-format\nblack\nisort[pyproject]>=4.3.15',
+        'deps': 'mypy==0.761\nflake8\nflake8-bugbear\nflake8-logging-format\nblack\nisort[pyproject]>=4.3.15',
```

Comment:
Can we use a range?

Reply:
Hmm, probably. I'm not even sure we *need* to pin. I've never encountered regressions from using mypy for over a year across projects now…

So maybe only pin our starting point?

```
mypy>=0.761
```

Reply:
Yes please